circle.yml

```diff
@@ -28,8 +28,11 @@ dependencies:
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
       conda install -q ciocheck -c spyder-ide --no-update-deps &&
-      if [ "$CIRCLE_NODE_INDEX" = "2" ]; then pip install -q pyqt5 spyder tornado pexpect; else conda install -q spyder tornado pexpect; fi &&
-      python setup.py build_static &&
+      if [ "$CIRCLE_NODE_INDEX" = "2" ]; then pip install -q pyqt5 tornado pexpect; else conda remove -q -y spyder && conda install tornado pexpect; fi &&
```

Comment:
`conda install -q -y spyder` in this line was fine. Please revert your previous change.

What you need to do is to add

    if [ "$CIRCLE_NODE_INDEX" = "2" ]; then pip uninstall -q -y spyder; else conda remove -q -y spyder; fi &&

below this line. Just that :-)

This removes the stable version installed in the previous step, so we can install the development one without surprises (e.g. having different files or modules from the two versions).